### DISCUSSION TITLE
graph-learn的进度较缓，后续计划是什么？

### DISCUSSION THREAD

**Discussion by HK-13** [alibaba/GraphScope#2072] [Q&A] [answered]
目前graphscope在分析和交互引擎的研发比较频繁，但是图学习引擎似乎进度较缓，能否说明下后续graph-learn的相关计划？

**Comment by LiSu** [comment, marked as answer]
Hi,

We just released Dynamic-Graph-Service (details in[ ref](https://graph-learn.readthedocs.io/en/latest/en/dgs/intro.html)) in GLE. 
New features like supports for dynamic graph GNN and distributed training on PyTorch are under development, which we plan to release in this November. We also plan to integrate GLE with PyG such that PyG models can be seamlessly trained using GLE in a distributed approach. You are always welcome to contact us with any problems/requirements/proposals for GLE.
